src/Microsoft.TestPlatform.CrossPlatEngine/Client/Parallel/ParallelOperationManager.cs

```diff
@@ -67,7 +67,6 @@ protected ParallelOperationManager(Func<T> createNewManager, int parallelLevel,
         public void RemoveManager(T manager)
         {
             this.concurrentManagerHandlerMap.Remove(manager);
-            this.DisposeInstance(manager);
```

Comment:
Remove the implementation and all consumers.

Reply:
resolved